plugins/conf_remap/CMakeLists.txt

```diff
@@ -17,3 +17,5 @@
 
 add_atsplugin(conf_remap conf_remap.cc)
 verify_remap_plugin(conf_remap)
+
+target_link_libraries(conf_remap PRIVATE libswoc::libswoc yaml-cpp::yaml-cpp)
```

Comment:
I see that conf_remap uses libswoc, but where is the yaml-cpp dependency coming from?

Reply:
It comes from this include

```
#include "tsutil/YamlCfg.h"
```